src/routes/v2/authenticated/__tests__/Sites.spec.js

```diff
@@ -71,7 +71,7 @@ describe("Sites Router", () => {
 
       expect(resp.body).toStrictEqual({ lastUpdated })
       expect(mockSitesService.getLastUpdated).toHaveBeenCalledWith(
-        mockUserWithSiteSessionData
+        mockUserWithSiteSessionData.siteName
```

Comment:
was this passing last time ._.

Reply:
Reverted this change in 3658486, I misunderstood how the accessToken is obtained for email users.